tests/integration_tests/test_matchmaker.py

```diff
@@ -303,7 +303,7 @@ async def test_game_matchmaking_disconnect(lobby_server):
 
     msg = await read_until_command(proto2, "match_cancelled", timeout=120)
 
-    assert msg == {"command": "match_cancelled"}
+    assert msg == {"command": "match_cancelled", "game_id": 41956}
```

Comment:
Where is this id set?

Reply:
It should be 1 higher than the highest game id in the test data.